### PR TITLE
Add a script to calculate a user's storage size from the metadata...

### PR DESCRIPTION
**Pull request by gregkare**
... And write it to Redis

Usage:

    ENVIRONMENT=development ./migrate_storage_size_from_metadata.rb username

### CODE REVIEW

**Comment by gregkare** [comment]
@galfert Can you try this one in your VM?

To delete the current key for your user you can use this command:

```
$ redis-cli del rs:s:galfert
```

File: migrate_storage_size_from_metadata.rb [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+          end
+        end
+      end
+
+      get_size_from_items("", "") -- Start from the root
+
+      redis.call("set", size_key, total_size)
+    EOF
+
+    redis.eval(lua_script, ["rs:s:#{user}", "test"], [user])
```
Comment (galfert) [inline migrate_storage_size_from_metadata.rb]: What's this "test" argument for?

**Comment by galfert** [comment]
When running the app in my VM I get the following error in the log:

> ERROR -- galfert: Error setting storage size from metadata for 'galfert': ERR Error running script (call to f_cdf0705163629910e2312e4cfe1a7eb73eb30500): @user_script:24: user_script:24: attempt to perform arithmetic on local 'file_size' (a boolean value)

**Comment by galfert** [comment]
Looks like I had some bogus Redis entry in my VM. Removed it and now the scripts works for me.

Generated size also looks good.

**Comment by gregkare** [comment]
I have removed the test key from the script, I was using it for debugging when I was writing the script

**Comment by galfert** [comment]
LGTM.

**Comment by gregkare** [comment]
Alright, let's merge this and run it on staging

**Comment by gregkare** [comment]
I ran it on staging (and ran it again after setting my storage size in Redis to 0 to confirm it works)

**Pull request merged**